src/parser/jobs/brd/SongUptime.js

```diff
@@ -41,17 +51,17 @@ export default class SongUptime extends Module {
 	}
 
 	_onComplete() {
+		this._setTolerance()
+		const songlessTime = this.util.formatDecimal((this._getSonglessTime())/1000)
+		const songlessTolerance = this._songlessTolerance / 1000
 
-		const fightDuration = (this.parser.fightDuration - this.downtime.getDowntime())/1000
-		const songlessTime = (this._getSonglessTime())/1000
-		const songlessPercentile = (songlessTime/fightDuration)*100
-
-		//TODO: Define a threshold for song uptime
-		if (songlessPercentile > 3) {
+		if (songlessTime > songlessTolerance) {
 			this.suggestions.add(new Suggestion({
 				icon: ACTIONS.THE_WANDERERS_MINUET.icon,
-				why: `Being songless for ${songlessTime} seconds (${songlessPercentile}%)`,
-				severity: songlessPercentile > 7 ? SEVERITY.MAJOR : songlessPercentile > 5? SEVERITY.MEDIUM : SEVERITY.MINOR,
+				why: <Fragment>
+					Being songless for {songlessTime} seconds.
```

Comment:
Consider updating the `why` to be a more natural "response" - something like "You were songless for...".